mbs_results/apply_estimation.py

```diff
@@ -0,0 +1,59 @@
+import glob
+
+import pandas as pd
+
+from mbs_results.calculate_estimation_weights import (
+    calculate_calibration_factor,
+    calculate_design_weight,
+)
+from mbs_results.pre_processing_estimation import get_estimation_data
+
+# from mbs_results.validate_estimation import validate_estimation
+
+
+def apply_estimation(population_path, sample_path, period, **config):
```

Comment:
type hints?

Reply:
Me and Giulia have both been having problems trying to run code with type hints but it's not consistent and I don't know why it's happening